### ISSUE TITLE
Add in a rewind option for the visualiser

### ISSUE THREAD

**Issue by JamieMair** [JamieMair/MuJoCo.jl#42]
Add in something similar to the following code to `ext/VisualiserExt/defaulthandlers.jl`:

```julia
onevent(
    KeyEvent,
    when = describe(GLFW.KEY_LEFT),
    what = "Step backwards when paused (hold SHIFT for $(SHIFTSTEPSPERKEY) steps)"
) do s, ev
    if ui.paused && ev.key === GLFW.KEY_LEFT && ispress_or_repeat(ev.action)
        steps = s.shift ? 50 : 1
        for _ = 1:steps
            reversestep!(p, mode(e))
        end
    end
end
```

**Comment by nic-barbara** [comment]
This only makes sense to implement once we've added a `Trajectory` mode for the visualiser (https://github.com/JamieMair/MuJoCo.jl/issues/41). In many cases it won't make sense (or be possible) to actually rewind the simulator or step it backwards in time unless we've saved a bunch of simulated states in a trajectory. The same is true for the reverse mode mentioned in https://github.com/JamieMair/MuJoCo.jl/issues/43.
